db/migrate/20211224092829_create_sites.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSites < ActiveRecord::Migration[6.1]
+  def change
+    create_table :sites do |t|
+      t.string :name, null: false
```

Comment:
@nanako0707 

```suggestion
      t.string :title, null: false
```

私が無知だったのですが、RSSのサイトは `title` というのが一般的なようだったので、修正していただけますか？ 🙏
ちょっと parse する時に面倒がありそうなので...

Reply:
了解しました！
修正します！